package.json

```diff
@@ -32,18 +31,15 @@
   },
   "homepage": "https://github.com/visualjerk/quill-magic-url",
   "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-stage-1": "^6.24.1",
-    "eslint": "4.14.0",
-    "webpack": "^4.14.0",
-    "webpack-cli": "^3.0.8",
-    "webpack-dev-server": "^3.1.4"
+    "@babel/core": "^7.7.4",
+    "babel-preset-env": "^1.7.0",
+    "webpack": "^4.41.2",
+    "webpack-cli": "^3.0.8"
   },
   "dependencies": {
-    "babel-loader": "^7.1.4",
-    "normalize-url": "^3.0.1",
+    "@babel/preset-env": "^7.7.4",
+    "babel-loader": "^8.0.6",
```

Comment:
I put babel-loader as a dependency by mistake. Please put babel-loader and @babel/preset-env inside devDependencies instead.